docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst

```diff
@@ -12,6 +12,11 @@ Release notes: Open GEE 5.3.2
 
       .. rubric:: New Features
 
+      **Documentation overhaul**. All documentation were migrated to be

```

Comment:
All documentation was migrated to Sphinx format, making it easier to edit and maintain. User documentation file names were also migrated from a number-based naming convention to self-explanatory descriptive titles.

Reply:
change was now made.